suites/pacific/cephadm/tier-0_remove_cluster.yaml

```diff
@@ -0,0 +1,95 @@
+#===============================================================================================
+#-------------------------------------
+#-----    Smoke Test suite     ------
```

Comment:
I think we don't need this suite as Ameena suggested we can have this `remove` test in an existing suite at the end.

Reply:
@sunilangadi2 yes, we can move this TC to tier-0 test suite, but I don't see any sense in adding this TC there. I would like to have other contributors opinion here.

Reply:
I concur with Sunil Angadi and Ameena's suggestions. purging is not a tier-0 operation. We could move it to a later tier to an existing test case. No need to have a new test case for this.

Reply:
Moved this TC to suite `suites/pacific/cephadm/tier-1_5-1_service-apply-spec.yaml`